### PR TITLE
add PubSub javascript

### LINKED ISSUE #305: install jQuery pub/sub so that click events are emitting global events
For image viewer:

https://github.com/mroderick/PubSubJS

### PR DESCRIPTION
**Pull request by mejackreed**
Closes #305

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
   ],
   "dependencies": {
     "listjs": "~1.1.1",
-    "openseadragon": "~1.2.0"
+    "openseadragon": "~1.2.0",
```
Comment (aeschylus) [inline bower.json]: Don't we want osd 2.0 in this viewer as well?

Comment (mejackreed) [inline bower.json]: Already done in #230

Comment (aeschylus) [inline bower.json]: Okay, I merged that one first (earlier today). I hope that doesn't cause a problem.

**Comment by aeschylus** [comment]
Yeah, now this has conflicts.
I'll go in and fix them.

**Comment by mejackreed** [comment]
@aeschylus Should be good to go :shipit:

**Comment by aeschylus** [comment]
Looks good locally. Merging.

**Pull request merged**